### PR TITLE
Update C++ tagging examples.

### PR DESCRIPTION
**Pull request by g-easy**
Fixes https://github.com/census-instrumentation/opencensus-website/issues/488

### CODE REVIEW

**Comment by g-easy** [comment]
cc @StevenYCChou

File: content/tag/value.md [hunk 1]
```diff
@@ -36,14 +36,12 @@ TagValue value = TagValue.create("memcache.Client.Get");
 {{</highlight>}}
 
 {{<highlight cpp>}}
-#include "opencensus/tags/tag_key.h"
-
-static const opencensus::tags::TagKey key_method =
-                    opencensus::tags::TagKey::Register("method");
+ABSL_CONST_INIT const absl::string_view kGetMethod =
```
Comment (StevenYCChou) [inline content/tag/value.md]: add #include "absl/strings/string_view.h" so the readers can directly use the example.

File: content/tag/map.md [hunk 2]
```diff
@@ -50,9 +50,13 @@ TagContext tagCtx = TAGGER.currentBuilder().
 
 {{<highlight cpp>}}
 #include "opencensus/tags/tag_key.h"
+#include "opencensus/tags/tag_map.h"
 
-static const opencensus::tags::TagKey key_method =
-                    opencensus::tags::TagKey::Register("method");
+void Get() {
+  static const auto method = opencensus::tags::TagKey::Register("method");
+  opencensus::tags::TagMap tm({{method, "memcache.Client.Get"}});
```
Comment (StevenYCChou) [inline content/tag/map.md]: as the reading flow from key -> value -> map, it makes sense for user to use the same way to declare value for TagMap.

saying
```
ABSL_CONST_INIT const absl::string_view kGetMethod =
    "memcache.Client.Get";

void Get() {
  static const auto method = opencensus::tags::TagKey::Register("method");
  opencensus::tags::TagMap tm({{method, kGetMethod}});
}
```

and it still makes sense to provide example to use `TagMap::TagMap(std::vector<std::pair<TagKey, std::string>> tags)`, so you can also provide it as 2nd example.

File: content/tag/value.md (hunk 1 above)
Comment (g-easy) [reply content/tag/value.md]: Done. Also, this is the most thorough way to declare a string_view constant. A user could just use a string literal, a std::string, etc.

File: content/tag/map.md (hunk 2 above)
Comment (g-easy) [reply content/tag/map.md]: Done and done. I also tried compiling it. :)

**Comment by g-easy** [comment]
PTAL.

**Comment by odeke-em** [comment]
Cool, thank you @StevenYCChou @g-easy!

**Pull request merged**